mbuild/utils/reference/oplsaa.xml

```diff
@@ -0,0 +1,3914 @@
+<ForceField>
```

Comment:
Could you cut this down to just the relevant parameters for methane?

Also, you will need to add `*.xml` to the [manifest file](https://github.com/mosdef-hub/mbuild/blob/61f621d9f691bfe98d56af34733db45a8df5ff8f/MANIFEST.in#L1).